ui/src/utils/enum-localization-utils.ts

```diff
@@ -18,13 +18,15 @@ import { Operation } from 'publication/publication-model';
 
 export interface LocalizedEnum<T> {
     value: T;
-    name: string;
+    get name(): string;
 }
 
 function values<T>(keyBase: string, enumValues: T[]): LocalizedEnum<T>[] {
     return enumValues.map((v) => ({
         value: v,
-        name: i18n.t(`enum.${keyBase}.${v}`),
+        get name() {
+            return i18n.t(`enum.${keyBase}.${v}`);
+        },
```

Comment:
Aiemmin enumien `i18n.t()`-kutsut käytiin ajamassa jo heti geoviitteen alustuksen yhteydessä kun ne olivat globaaleja. Nyt ne on muutettu toimimaan typescriptin getter-mekanismilla, että i18next ehtii initialisoitua ennen kääntämisen yrittämistä

Reply:
Mitenkä tämä toimii käytännössä? Eikö namen getteriä voi kutsua ennen kuin i18n on ladannut bäkkäristä käännöstiedoston, jolloin se palauttais käännösavaimen?

Reply:
Joo, juurikin noin. Periaatteessa mikään ei estä etteikö näitä gettereitä voitaisi kutsua vaikka jostain globaalisti, mutta _käytännössä_ niin pitkään kun niitä kutsutaan vaan mistä tahansa Reactin ajamasta jutusta, niin kaikki on ok